app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -54,6 +67,7 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
 
     private val binding by viewBinding(ActivityLoggedInBinding::bind)
 
+    private val date = Date()
```

Comment:
We should use `LocalDate` from the `java.time.*`-API, and I don't think it needs to be a class-level field 🙂

Reply:
Isnt it good to have it class-level since its used in two funktions (`onCreateOptionsMenu` and `shouldShowTooltip`) so we only have one instance of it?

Reply:
It's alright to have it be locally used. If we're worried about instance re-use we can perhaps pass it between the functions. Class-level fields have a tendency to make things a lot more complicated over time, and hence we should avoid them unless entirely necessary 🙂